WooCommerce/Classes/ViewRelated/Blaze/AdDestination/BlazeAdDestinationSettingView.swift

```diff
@@ -8,72 +8,83 @@ struct BlazeAdDestinationSettingView: View {
 
     typealias DestinationType = BlazeAdDestinationSettingViewModel.DestinationURLType
 
+    @State private var isShowingAddParameterView = false
+
     init(viewModel: BlazeAdDestinationSettingViewModel) {
         self.viewModel = viewModel
     }
 
     var body: some View {
         NavigationView {
-            VStack(alignment: .leading, spacing: 0) {
-                // URL destination section
-                VStack(spacing: 0) {
-                    sectionHeading(title: Localization.destinationUrlHeading)
+            List {
+                Section {
                     destinationItem(title: Localization.productURLLabel,
                                     subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.productURL),
-                                    type: DestinationType.product,
-                                    showBottomDivider: true)
+                                    type: DestinationType.product)
+                    .listRowInsets(EdgeInsets())
 
                     destinationItem(title: Localization.siteHomeLabel,
                                     subtitle: String(format: Localization.destinationUrlSubtitle, viewModel.homeURL),
                                     type: DestinationType.home)
+                    .listRowInsets(EdgeInsets())
+                } header: {
+                    Text(Localization.destinationUrlHeading)
                 }
-                .padding(.bottom, Layout.sectionVerticalSpacing)
-
-                // URL parameters section
-                VStack {
-                    sectionHeading(title: Localization.urlParametersHeading)
 
-                    VStack {
-                        ForEach(viewModel.parameters, id: \.self) { parameter in
-                            parameterItem(itemName: parameter.key)
-                        }
 
-                        Button(Localization.addParameterButton) {
-                            // todo
+                Section {
+                    ForEach(viewModel.parameters, id: \.id) { parameter in
```

Comment:
Nit: you can remove `id` parameter if the items in the list conform to `Identifiable`.

Reply:
TIL! That's convenient.  Added in https://github.com/woocommerce/woocommerce-ios/pull/11642/commits/26df3c7a824e464fc2896cf3a76f1868e89aecd0